auditlog_ext_nrq/tests/test_auditlog_ext_nrq.py

```diff
@@ -64,3 +64,45 @@ def test_03_unlink_record(self):
             ('res_id', '=', self.translation_record.id),
         ], limit=1, order='id desc')
         self.assertEqual(log.log_category, 'unlink')
+
+    def test_04_create_vals(self):
+        """
+            Test old value text,new value text with datetime field data
+            Test old value text,new value text without datetime field data
+        """
+        tz = 'Japan'
+
+        # Default timezone for super user is Europe/Brussels so change to Japan
+        self.env.user.write({'tz': tz})
```

Comment:
```suggestion
        # Japan is 9 hours ahead of UTC.
        self.env["res.users"].browse(SUPERUSER_ID).write({"tz": "Japan"})
```

I think it's better to be explicit that the timezone is being set to the superuser here.